scripts/integration-test-all.sh

```diff
@@ -3,31 +3,35 @@
 # "stable" mode tests assume data is static
 # "live" mode tests assume data dynamic
 
-
-SCRIPT=`basename ${BASH_SOURCE[0]}`
+SCRIPT=$(basename ${BASH_SOURCE[0]})
 TEST=""
+QTD=1
+TIMEOUT=5
 
-PORT="7001"
-RPC_PORT="8545"
+PORT="700"
+RPC_PORT="854"
```

Comment:
because its for testing and you dont want to conflict with actual running instances

Reply:
I guess by standard I meant four digit port numbers. I am well aware that you are avoiding conflicting ports when running instances.

Certain operating systems may restrict opening lower port numbers, so it is usually a standard to use four digit port numbers. The numbers you are using also make it look like someone accidentally deleted the last digit off our default ports. It might be nice to make this more distinct and to use a more common port number or just add 1 to the last digit.

Reply:
yeah i think port numbers < 1024 are reserved for the kernel (at least depending on OS) change to 8546 or sometihng?

Reply:
I see you guys are not familiar with how the shell scripts work, but that is the initial prefix for the ports. You can see later on that the $PORT and $RPC_PORT is being used with adding a suffix for the port.

Reply:
Ok, I see. Please add comments in the code here to better document this. Seems like it might be an easy mistake for people to make.

Reply:
oks